modules/calib3d/test/test_usac.cpp

```diff
@@ -361,8 +358,8 @@ TEST_P(usac_Essential, maxiters) {
                 FAIL() << "Essential matrix estimation failed!\n";
             else continue;
         }
-        EXPECT_NE(roll_results_sum, 0);
     }
+    EXPECT_NE(roll_results_sum, 0);
```

Comment:
It's not obvious that `roll_results_sum += cv::norm(E1, E2, NORM_L1) != 0;` happens even once. Previous line may hide actual behavior.  We need to check amount of successful `findEssentialMat` at least.

Reply:
I have not written this test, and I do not think it is necessary. As I understand, it checks that output of RANSAC is not the same for 1 and 1000 iterations. However, given random behaviour of RANSAC it could happen that the output will be the same.